ui/components/app/edit-gas-fee-popover/network-statistics/status-slider/status-slider.js

```diff
@@ -24,24 +25,24 @@ const determineStatusInfo = (givenNetworkCongestion) => {
   const color = GRADIENT_COLORS[colorIndex];
   const sliderTickValue = colorIndex * 10;
 
-  if (networkCongestion <= 0.33) {
```

Comment:
I changed this slightly so that instead of:

* not busy: <= 0.33
* stable: > 0.33, <= 0.66
* busy: > 0.66

It is now:

* not busy: 0 up to 0.33
* stable: 0.33 up to 0.66
* busy: 0.66-

This is not strictly necessary to change, but given that I need to _just_ test that the network is busy elsewhere, it made more sense in my head to say that the status changes to busy as soon as `networkCongestion` crosses 0.66 instead of as soon as it exceeds it. Happy to change this back though.